ros2_control_demo_bringup/launch/rrbot_gazebo.launch.py

```diff
@@ -14,9 +14,8 @@
 
 from launch import LaunchDescription
 from launch.actions import IncludeLaunchDescription
-from launch.substitutions import Command, FindExecutable, PathJoinSubstitution
 from launch.launch_description_sources import PythonLaunchDescriptionSource
-
+from launch.substitutions import Command, FindExecutable, PathJoinSubstitution
```

Comment:
Naive question: why not keeping the same order between the two files rrbot_gazebo.launch.py and rrbot_ignition.launch.py ?

Not very important but just making things looking the same among the examples.

Reply:
Yes, we should definitely do that!